package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@equinor/amplify-components",
-  "version": "7.4.3",
+  "version": "7.5.0",
```

Comment:
I think you need to run ‘npm i‘ to update the ‘package-lock‘ as well, build will fail if not IIRC